--- vendor/danog/madelineproto/src/MTProtoTools/UpdateHandler.php
+++ vendor/danog/madelineproto/src/MTProtoTools/UpdateHandler.php
@@ -384,6 +384,17 @@ trait UpdateHandler
             }
         }
     }
+	public function cancelGetUpdates(): bool
+	{
+		if (!$this->usingGetUpdates || !$this->update_deferred) {
+			return false;
+		}
+		
+		$deferred = $this->update_deferred;
+		$this->update_deferred = null;
+		$deferred->error(new \Exception("cancel"));
+		return true;
+	}
     /**
      * Check message ID.
      *
